docs/django-widget.rst

```diff
@@ -56,7 +93,52 @@ widget url:
 
     UPLOADCARE = {
         # ...
-        'upload_base_url' = 'http://path.to/your/upload/handler',
+        "upload_base_url": "http://path.to/your/upload/handler",
+    }
+
+Use ``widget_options`` to pass arbitrary `options`_ to the file uploader:
+
+.. code-block:: python
+
+    UPLOADCARE = {
+        # ...
+        "widget": {
+            "options": {
+                "source-list": "local,url,camera",
+                "camera-mirror": True,
+            },
+        },
+    }
+
+
+.. _django-legacy-widget-settings-ref:
+
+Settings for legacy widget
+--------------------------
+
+If you want to use our legacy jQuery-widget, you can enable it in settings:
+
+.. code-block:: python
+
+
+    UPLOADCARE = {
+        "pub_key": "demopublickey",
+        "secret": "demoprivatekey",
```

Comment:
It the example above you omit demo keys, and here you keep them. Is this on purpose?

Reply:
Yes.

The block above is not an example, it's the default settings. But these default settings will give you errors when you try to run the project. So you'll have to specify at least something in order it to run. Like in an example you're referring to here.